docs/extensions/guides/renderer-extension.md

```diff
@@ -78,14 +90,15 @@ export class ExamplePage extends React.Component<{ extension: LensRendererExtens
 }
 ```
 
-Note that the `ExamplePage` class defines a property named `extension`.
-This allows the `ExampleExtension` object to be passed in React-style in the cluster page definition, so that `ExamplePage` can access any `ExampleExtension` subclass data.
+Note that the `ExamplePage` class defines the `extension` property. This allows the `ExampleExtension` object to be passed in the cluster page definition in the React style. This way, `ExamplePage` can access all `ExampleExtension` subclass data.
+
+The above example shows how to create a cluster page, but not how to make that page available to the Lens user. Use `clusterPageMenus`, covered in the next section, to add cluster pages to the Lens UI.
 
 ### `clusterPageMenus`
 
-The above example code shows how to create a cluster page but not how to make it available to the Lens user.
-Cluster pages are typically made available through a menu item in the cluster dashboard sidebar.
-Expanding on the above example a cluster page menu is added to the `ExampleExtension` definition:
+`clusterPageMenus` allows you to add cluster page menu items to the left side menu.
```

Comment:
You use "left side menu" for both cluster pages and global pages. Cluster page menus go in the cluster dashboard, on the left side of the dashboard, to the right of the cluster menu. Global pages go below the cluster menu, to the left of the dashboard UI. Does that make sense? Need a good name for each...

Reply:
Thanks, Jim. Let's try Global pages: left nav, Cluster page: secondary left nav.